utils/data-labels.ts

```diff
@@ -0,0 +1,17 @@
+import type { IssueStatus } from '~/constants/issues'
```

Comment:
Very happy to have a second pair of eyes on this file, I've gone through a few version of it and ended up with this.

Reply:
Addressed in other discussions.